docs/reference/api/docker_remote_api_v1.21.md

```diff
@@ -2681,10 +2738,10 @@ Status Codes:
 
 JSON Parameters:
 
-- **name** - The new network's name. this is a mandatory field
-- **driver** - Name of the network driver to use. Defaults to `bridge` driver
-- **options** - Network specific options to be used by the drivers
-- **check_duplicate** - Requests daemon to check for networks with same name
+- **Name** - The new network's name. this is a mandatory field
+- **Driver** - Name of the network driver to use. Defaults to `bridge` driver
+- **Options** - Network specific options to be used by the drivers
+- **CheckDuplicate** - Requests daemon to check for networks with same name
```

Comment:
@thaJeztah good point. we might be missing information on IPAM Driver in the docs. I will work with @moxiegirl on that.